storagenode/gracefulexit/worker.go

```diff
@@ -151,8 +156,17 @@ func (worker *Worker) transferPiece(ctx context.Context, transferPiece *pb.Trans
 		return err
 	}
 
-	putCtx, cancel := context.WithCancel(ctx)
-	defer cancel()
+	putCtx := ctx
+
+	if worker.minBytesPerSecond > 0 {
```

Comment:
Should we return an error if this configuration is not set? Is there a default timeout for the connection?

Reply:
`minBytesPerSecond` has a default value of 128 bytes. I don't think we need to return an error if it's not set, but I'm not positive. the `maxTransferTime` will be `pieceSize`/`minBytesPerSecond`. I'm not sure what the default pieceSize is right now though.

Reply:
What if a "bad" storagenode set its `minBytesPerSecond` to 0, then there will be no timeout for their transfer?

Reply:
you're right- I added a check that sets it to 128 bytes if it's set to 0